eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,93 @@
+$Language = "rust"
+$LanguageDisplayName = "Rust"
+$PackageRepository = "crates.io"
+$packagePattern = "Cargo.toml"
+$MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/main/_data/releases/latest/rust-packages.csv"
+$GithubUri = "https://github.com/Azure/azure-sdk-for-rust"
+$PackageRepositoryUri = "https://crates.io/crates"
+
+function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
```

Comment:
```suggestion
function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) 
{
```
nit: I generally like `{` on a newline for larger blocks.

Reply:
With autoformat on save turned on, this gets reverted every time the file's edited.   The options for braces aren't granular enough to have it use "next line" for function definitions and "same line" for other blocks

Reply:
I prefer consistency as well. Don't fight the formatter. It's a losing battle.